code/AEPSDK.brs

```diff
@@ -203,24 +203,34 @@ function AdobeAEPSDKInit(taskNode = invalid as dynamic) as object
         '
         ' This function allows passing custom identifiers using identityMap.
         '
-        ' @param data as object : xdm data
+        ' @param data as object : an associative array containing XDM and/or free-form data
         ' @param [optional] callback as function(context, result) : handle Edge response
         ' @param [optional] context as dynamic : context to be passed to the callback function
         '
         ' *************************************************************************************
 
-        sendEvent: function(xdmData as object, callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
+        sendEvent: function(data as object, callback = _adb_defaultCallback as function, context = invalid as dynamic) as void
             _adb_logDebug("API: sendEvent()")
-            if _adb_isEmptyOrInvalidMap(xdmData) then
-                _adb_logError("sendEvent() - Cannot send event, invalid XDM data")
+
+            if _adb_isEmptyOrInvalidMap(data) then
+                _adb_logError("sendEvent() - Invaid event data, must be an associative array")
```

Comment:
Typo - Invalid
nit: Can you make this consistent with other log messages in this function?